lib/octicons_gem/test/octicons_test.rb

```diff
@@ -4,9 +4,8 @@
   it "loads all icons on initialization" do
     refute_equal 0, OcticonsV2::OCTICON_SYMBOLS.length
     x_icon = OcticonsV2::OCTICON_SYMBOLS["x"]
+    assert x_icon["name"]
     assert x_icon["keywords"]
-    assert x_icon["path"]
-    assert x_icon["height"]
-    assert x_icon["width"]
+    assert x_icon["heights"]
```

Comment:
Might we add a deep assertion for the content of heights?

Reply:
Is there a good way to make deep assertions on `heights` without making assumptions about what keys it has?

Reply:
The type definition for `heights` would look something like this:

```ts
heights: {
  [key: string]: {
    width: number
    path: string
  }
}
```